tests/test_broker.py

```diff
@@ -3,6 +3,9 @@
 
 from regolith.broker import Broker, load_db
 
+import tempfile
+
+tmpdir = tempfile.gettempdir()
```

Comment:
Global variables should be upercase, so `TMPDIR = ...`

Reply:
done